addons/website_event_booth/static/src/js/booth_register.js

```diff
@@ -65,11 +72,63 @@ publicWidget.registry.boothRegistration = publicWidget.Widget.extend({
         });
     },
 
+    /**
+     * Check if the confirmation form is valid by testing each of its inputs
+     *
+     * @private
+     * @param $form
+     * @return {boolean} - true if no errors else false
+     */
+    _isConfirmationFormValid($form) {
+        const formErrors = [];
+
+        $form.find('.form-group').each(function (index, field) {
+            let $field = $(field);
+            let inputs = $field.find('.form-control');
+            let invalidInputs = inputs.toArray().filter(function (input) {
```

Comment:
I think jQuery supports a "filter" where you can pass a function so you don't have to transform it into an array first.

https://api.jquery.com/filter/